### ISSUE TITLE
[Feature Request] set manually location for refuel

### ISSUE THREAD

**Issue by k4r3l01** [Stephan-S/FS19_AutoDrive#2011]
Hello please its will be possible to add option to set point for refueling instead of automatic searching for refueling because even if i have path through fuel trigger so AD say me could not find refuel station for nafta

**Comment by Axel32019** [comment]
Hello @k4r3l01 , Could you please provide a screenshot of the fuel trigger AD could not find.

**Comment by k4r3l01** [comment]
trigger is marked on this picture

![fsScreen_2021_07_09_23_30_19](https://user-images.githubusercontent.com/6354127/125138234-91fcfd00-e10e-11eb-9886-51bc8a8e1767.png)

**Comment by k4r3l01** [comment]
map is JZD Vidhostice

**Comment by Axel32019** [comment]
Did you set a destination ~2..3 Waypoints behind?

**Comment by k4r3l01** [comment]
yep here is screenshot
![fsScreen_2021_07_10_00_09_02](https://user-images.githubusercontent.com/6354127/125140609-308b5d00-e113-11eb-84bc-6c682876f5dd.png)

**Comment by Axel32019** [comment]
Which AD version you are using?
Could you refuel manuel?

**Comment by k4r3l01** [comment]
manual refuel working withoud AD and version i have 1.1.1.4

**Comment by k4r3l01** [comment]
and about this problem this refuel location is by GC and strange think is today find refuel station but only on refinery for diesel

**Comment by k4r3l01** [comment]
and on refinery is trigger only for cistern not for direct refuel for cars etc.

**Comment by Axel32019** [comment]
Hello @k4r3l01 , This issue is related to a GC loading trigger issue - fixed in alpha branch.
No need for manual location to be set.

**Comment by k4r3l01** [comment]
i dont think so im now on alpha build and still my drivers ignoring station on my farm but i dont know if problem is not that waipoint is after AdBlue station  because in my case is fuel station->adblue tation->waypoint

**Comment by k4r3l01** [comment]
here is video driver really ignoring fuel station on my farm and on video is version of AD 1.1.1.5.RC1
https://youtu.be/nH0IRep9m5U

**Comment by Axel32019** [comment]
Could you please attach the following files, so I could have a look at it.
AutoDrive_config.xml, items.xml, onCreateObjects.xml, vehicles.xml

**Comment by Axel32019** [comment]
@k4r3l01 Any update?
